util/ibex_config.py

```diff
@@ -214,6 +214,46 @@ def get_config_file_location():
     return os.environ.get('IBEX_CONFIG_FILE', _DEFAULT_CONFIG_FILE)
 
 
+def parse_config(config_name, config_filename=get_config_file_location()):
```

Comment:
I think that both call sites set `config_filename`, so I'd suggest removing the default argument.

Reply:
done